modules/statics/src/coins.ts

```diff
@@ -476,6 +477,7 @@ export const coins = CoinMap.fromCoins([
   erc20('gold', 'XBullion Token', 8, '0x0e573fdd3c5acf3a74f1775b0f9823a1a0e2b86b', UnderlyingAsset.GOLD),
   erc20('got', 'GOExchange', 18, '0xf11f2550769dac4226731b7732dd4e17e72b1b01', UnderlyingAsset.GOT),
   erc20('grt', 'The Graph', 18, '0xc944e90c64b2c07662a292be6244bdf05cda44a7', UnderlyingAsset.GRT),
+  erc20('gtc', 'Gitcoin', 18, '0xde30da39c46104798bb5aa3fe8b9e0e1f348163f', UnderlyingAsset.GTC),
```

Comment:
Are you using `Contract Overview` or `Token Tracker` for the names here? There looks to be inconsistencies between coin names in the PR.

Reply:
I think we are just going by what is mentioned in the Jira ticket. @doodles92 can you confirm if the changes made are accurate

Reply:
This is correct; confirmed